src/controllers/hid/hiddenylist.h

```diff
@@ -1,21 +1,35 @@
 #pragma once
 
-typedef struct hid_denylist {
+// TODO: unify this with the invalid interfacenumber from the bulkenumerator
+constexpr static int kInvalidInterfaceNumber = -1;
+constexpr static unsigned short kBlanketValue = 0x0;
+
+struct hid_denylist_t {
     unsigned short vendor_id;
     unsigned short product_id;
     unsigned short usage_page;
     unsigned short usage;
-    int interface_number;
-} hid_denylist_t;
+    int interface_number = kInvalidInterfaceNumber;
+};
 
 /// USB HID device that should not be recognized as controllers
-constexpr hid_denylist_t hid_denylisted[] = {
-        {0x1157, 0x300, 0x1, 0x2, -1},  // EKS Otus mouse pad (OS/X,windows)
-        {0x1157, 0x300, 0x0, 0x0, 0x3}, // EKS Otus mouse pad (linux)
-        {0x04f3, 0x2d26, 0x0, 0x0, -1}, // ELAN2D26:00 Touch screen
-        {0x046d, 0xc539, 0x0, 0x0, -1}, // Logitech G Pro Wireless
+constexpr static hid_denylist_t hid_denylisted[] = {
+        {0x1157, 0x300, 0x1, 0x2},                          // EKS Otus mouse pad (OS/X,windows)
+        {0x1157, 0x300, kBlanketValue, kBlanketValue, 0x3}, // EKS Otus mouse pad (linux)
+        {0x04f3, 0x2d26, kBlanketValue, kBlanketValue},     // ELAN2D26:00 Touch screen
```

Comment:
allowing to omit the interface_number but not the usage_page and usage is not consistent. 
I think I would prefer always pass all values and revert the default in hid_denylist_t. But we can decide also the other way around.

Reply:
> allowing to omit the interface_number but not the usage_page and usage is not consistent.

why?